chia/util/keychain.py

```diff
@@ -437,6 +437,11 @@ def delete_key_by_fingerprint(self, fingerprint: int) -> int:
             try:
                 key_data = self._get_key_data(index, include_secrets=False)
                 if key_data.fingerprint == fingerprint:
+                    try:
+                        self.keyring_wrapper.delete_label(key_data.fingerprint)
+                    except Exception:
```

Comment:
I'm channeling my inner-Kyle here, but should we be catching a specific exception, like `KeychainUserNotFound`?

Reply:
If so it would rather be `KeychainError` but i decided to not because we two lines below also catch `Exception` when deleting the key. Anyway, you got me :)